src/opnsense/mvc/app/controllers/OPNsense/Routes/forms/dialogEditRoute.xml

```diff
@@ -15,7 +15,7 @@
     <id>route.gateway</id>
     <label>Gateway</label>
     <type>dropdown</type>
-    <help><![CDATA[Choose which gateway this route applies to or <a href="/system_gateways_edit.php">add a new one</a>.]]></help>
+    <help><![CDATA[Choose which gateway this route applies to or <a href="/ui/gateways">add a new one</a>.]]></help>
```

Comment:
strings like this are never translated due to embedded html. better to ditch the link for long-term prosperity